.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 sudo: required
 dist: trusty
 python: '3.6'
+node: '10'
```

Comment:
Worth noting - I don't know if this works.  Part of the reason for creating the PR was to force it to run in travis.